rgw/v2/tests/s3_swift/test_bucket_lifecycle_object_expiration.py

```diff
@@ -64,6 +64,8 @@ def test_exec(config):
     for i in op:
         if i in ['nautilus','pacific']:
                 ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_lc_max_worker, str(config.rgw_lc_max_worker))
+        else:
+            utils.exec_shell_cmd("ceph config set client.rgw rgw_lc_debug_interval 30")
```

Comment:
Based on the value of `op`, I believe the `else` statement would be executed irrespective of the release being `pacific`. IMO, we could do this

```
if "pacific" in op:
  commands = [
    f"ceph config set client.rgw rgw_lc_max_worker {config.rgw_lc_max_worker}",
     "ceph config set client.rgw rgw_lc_debug_interval 30"
  ]
  for command in commands: utils.exec_shell_cmd(command)
else:
  ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_lc_max_worker, str(config.rgw_lc_max_worker)
```